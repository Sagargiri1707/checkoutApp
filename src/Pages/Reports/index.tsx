function Test(){
    return <div>
         <div className="container mx-auto w-128 ">
      <div className="mt-4">
        <h3>Test Case Reports</h3>
        <img src="https://rukminim1.flixcart.com/www/1500/1500/promos/30/01/2024/68190fe8-ee28-480c-bca2-d0b847e9bf53.png?q=100" alt="test case"/>
        <h3 className="mb-2 text-lg font-semibold text-gray-900 dark:text-white">
          Lighthouse insights
        </h3>
        </div>
        <div className="flex flex-col space-y-4 p-4">
                <div>Performance Tab</div>
              <div className="flex font-sm items-center">
                <span className="font-bold">
                    <img src="https://rukminim1.flixcart.com/www/1200/1200/promos/27/01/2024/fb7b7d53-121d-40f0-a8c9-b8a47944365e.png?q=100"/>
                </span>
              </div>
              <div>Accessibilty </div>

              <div className="flex font-sm items-center">
                <span className="font-bold">
                    <img src="https://rukminim1.flixcart.com/www/1200/1200/promos/27/01/2024/e2f0b17c-f3a5-4701-b898-145c9ebf8415.png?q=100"/>
                </span>
              </div>
              <div>Best Practice</div>

              <div className="flex font-sm items-center">
                <span className="font-bold">
                    <img src="https://rukminim1.flixcart.com/www/1200/1200/promos/27/01/2024/25f1f702-9964-404d-ba54-20bcc613cdfb.png?q=100"/>
                </span>
              </div>
              <div>SEO</div>

              <div className="flex font-sm items-center">
                <span className="font-bold">
                    <img src="https://rukminim1.flixcart.com/www/1200/1200/promos/27/01/2024/fcf46c3b-42fc-4856-ab0a-db789f6ed591.png?q=100"/>
                </span>
              </div>
        </div>
        </div>
    </div>
}

export default Test