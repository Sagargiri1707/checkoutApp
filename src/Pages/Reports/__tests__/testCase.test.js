import '@testing-library/jest-dom';
import React from 'react';
import { render, screen } from '@testing-library/react';
import Reports from '../index';

describe('navbar component', () => {
  it('should render a header ', () => {
   const container= render(<Reports />);
    expect(container).toMatchSnapshot();
  });
});
