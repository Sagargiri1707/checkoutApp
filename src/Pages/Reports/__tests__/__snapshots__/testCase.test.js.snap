// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar component should render a header  1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="container mx-auto w-128 "
        >
          <div
            class="mt-4"
          >
            <h3>
              Test Case Reports
            </h3>
            <img
              alt="test case"
              src="https://rukminim1.flixcart.com/www/1500/1500/promos/30/01/2024/68190fe8-ee28-480c-bca2-d0b847e9bf53.png?q=100"
            />
            <h3
              class="mb-2 text-lg font-semibold text-gray-900 dark:text-white"
            >
              Lighthouse insights
            </h3>
          </div>
          <div
            class="flex flex-col space-y-4 p-4"
          >
            <div>
              Performance Tab
            </div>
            <div
              class="flex font-sm items-center"
            >
              <span
                class="font-bold"
              >
                <img
                  src="https://rukminim1.flixcart.com/www/1200/1200/promos/27/01/2024/fb7b7d53-121d-40f0-a8c9-b8a47944365e.png?q=100"
                />
              </span>
            </div>
            <div>
              Accessibilty 
            </div>
            <div
              class="flex font-sm items-center"
            >
              <span
                class="font-bold"
              >
                <img
                  src="https://rukminim1.flixcart.com/www/1200/1200/promos/27/01/2024/e2f0b17c-f3a5-4701-b898-145c9ebf8415.png?q=100"
                />
              </span>
            </div>
            <div>
              Best Practice
            </div>
            <div
              class="flex font-sm items-center"
            >
              <span
                class="font-bold"
              >
                <img
                  src="https://rukminim1.flixcart.com/www/1200/1200/promos/27/01/2024/25f1f702-9964-404d-ba54-20bcc613cdfb.png?q=100"
                />
              </span>
            </div>
            <div>
              SEO
            </div>
            <div
              class="flex font-sm items-center"
            >
              <span
                class="font-bold"
              >
                <img
                  src="https://rukminim1.flixcart.com/www/1200/1200/promos/27/01/2024/fcf46c3b-42fc-4856-ab0a-db789f6ed591.png?q=100"
                />
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="container mx-auto w-128 "
      >
        <div
          class="mt-4"
        >
          <h3>
            Test Case Reports
          </h3>
          <img
            alt="test case"
            src="https://rukminim1.flixcart.com/www/1500/1500/promos/30/01/2024/68190fe8-ee28-480c-bca2-d0b847e9bf53.png?q=100"
          />
          <h3
            class="mb-2 text-lg font-semibold text-gray-900 dark:text-white"
          >
            Lighthouse insights
          </h3>
        </div>
        <div
          class="flex flex-col space-y-4 p-4"
        >
          <div>
            Performance Tab
          </div>
          <div
            class="flex font-sm items-center"
          >
            <span
              class="font-bold"
            >
              <img
                src="https://rukminim1.flixcart.com/www/1200/1200/promos/27/01/2024/fb7b7d53-121d-40f0-a8c9-b8a47944365e.png?q=100"
              />
            </span>
          </div>
          <div>
            Accessibilty 
          </div>
          <div
            class="flex font-sm items-center"
          >
            <span
              class="font-bold"
            >
              <img
                src="https://rukminim1.flixcart.com/www/1200/1200/promos/27/01/2024/e2f0b17c-f3a5-4701-b898-145c9ebf8415.png?q=100"
              />
            </span>
          </div>
          <div>
            Best Practice
          </div>
          <div
            class="flex font-sm items-center"
          >
            <span
              class="font-bold"
            >
              <img
                src="https://rukminim1.flixcart.com/www/1200/1200/promos/27/01/2024/25f1f702-9964-404d-ba54-20bcc613cdfb.png?q=100"
              />
            </span>
          </div>
          <div>
            SEO
          </div>
          <div
            class="flex font-sm items-center"
          >
            <span
              class="font-bold"
            >
              <img
                src="https://rukminim1.flixcart.com/www/1200/1200/promos/27/01/2024/fcf46c3b-42fc-4856-ab0a-db789f6ed591.png?q=100"
              />
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
